util/types/convert_test.go

```diff
@@ -40,6 +40,7 @@ type invalidMockType struct {
 func Convert(val interface{}, target *FieldType) (v interface{}, err error) {
```

Comment:
`Convert` is in test code, it's a wrap on `ConvertTo`.
So, what's new in this change:
1. something convert to timestamp
2. timestamp convert to something

Something ConvertTo timestamp is totally rewritten in function `convertToMysqlTimestamp`.
timestamp convert to something use the old logic. Both are simple enough.

I'm afraid just testing the Convert won't help the real case, I'd like to port some test case
from MySQL instead. @XuHuaiyu